Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -38,7 +38,7 @@ fileprivate let scaleOutTransformRight = CGAffineTransform.identity.translatedBy
  **/
 class PlaceDetailViewController: UIViewController {
 
-    weak var dataSource: PlaceDataSource? {
+    weak var dataSource: PlacesProvider? {
```

Comment:
`PlaceDataSource` wfm – I kept it as `PlacesProvider` for simplicity in not touching every file. I filed #528.